aws-lambda/src/http.rs

```diff
@@ -0,0 +1,56 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use hyper::body::Bytes;
+use reqwest::Client;
+use tailcall::http::Response;
+use tailcall::HttpIO;
+
+#[derive(Clone)]
+pub struct LambdaHttp {
+    client: Client,
+}
+
+impl Default for LambdaHttp {
+    fn default() -> Self {
+        Self { client: Client::new() }
+    }
+}
+
+impl LambdaHttp {
+    pub fn init() -> Self {
+        Default::default()
+    }
+}
+
+#[async_trait::async_trait]
+impl HttpIO for LambdaHttp {
+    async fn execute(&self, request: reqwest::Request) -> Result<Response<Bytes>> {
+        let client = self.client.clone();
+        let method = request.method().clone();
+        let url = request.url().clone();
+        let response = client.execute(request).await?.error_for_status()?;
+        let res = Response::from_reqwest(response).await?;
+        tracing::info!("{} {} {}", method, url, res.status.as_u16());
```

Comment:
since you're propagating errors and use `error_for_status` here in status you'll see only success statuses that might be not what you've intended

Reply:
I think removing `error_for_status` could be a problem because we will consider that request as successful and will try to parse the body and at the end will respond with nulls instead of erroring

Reply:
Oh, yeah. Same code is in `cloudflare`. Re-added.